src/components/Canvas/interactionLayer.tsx

```diff
@@ -0,0 +1,505 @@
+import { PixelChangeRecords } from "../../helpers/PixelChangeRecords";
+import {
+  addColumnToData,
+  addRowToData,
+  deleteColumnOfData,
+  deleteRowOfData,
+  extractColoredPixelsFromColumn,
+  extractColoredPixelsFromRow,
+  getColumnCountFromData,
+  getGridIndicesFromData,
+  getRowCountFromData,
+} from "../../utils/data";
+import { convertCartesianToScreen, getScreenPoint } from "../../utils/math";
+import { BaseLayer } from "./BaseLayer";
```

Comment:
Interesting! Such error does not occur in my environment. For now I will skip this change. It seems that it might be due to the version of `yarn`. My version is `1.22.10`